data/config_files/multi_dimensional_hierarchical_model.yml

```diff
@@ -0,0 +1,88 @@
+schema_version: 1
+
+# ----------------------------------------------------------------------
+model:
+  target: pymc_marketing.mmm.multidimensional.MMM
+  kwargs:
+    date_column: "date"
+    channel_columns: # explicit for reproducibility
+      - channel_1
+      - channel_2
+      # …
+    target_column: "kpi_registrations"
+    dims: ["market"]  # Keep as array for consistency
+    # Standard scaling format
+    scaling:
+      type: "max"
+      target:
+        method: "max"
+        dims: []
+      channel:
+        method: "max"
+        dims: []
+
+    # --- media transformations ---------------------------------------
+    adstock:
+      target: pymc_marketing.mmm.GeometricAdstock
+      kwargs: {l_max: 12}        # any other hyper-parameters here
+
+    saturation:
+      target: pymc_marketing.mmm.LogisticSaturation
+      kwargs:
+        priors:
+          beta:
+            distribution: "HalfNormal"
+            sigma: 1.0
+            dims: "channel"
```

Comment:
Why are there multiple forms to represent a distribution? This and below with "target'